web/views/view_helpers.ex

```diff
@@ -61,10 +61,14 @@ defmodule App.ViewHelpers do
     ~s(<a href="/#{data.content}/#{data.page}">#{text}</a>)
   end
 
-  def get_class(component) do
+  def get_class(component, background \\ "light") do
     case component do
       "primary_button" -> "f5 fw6 link dib ph3 pv2 br-pill lm-white-hover lm-dark-blue lm-bg-orange pointer button segoe-bold tracked"
-      "secondary_button" -> "f5 link dib ph3 pv2 br-pill lm-bg-dark-blue lm-white-hover b--lm-white-hover lm-orange pointer ba bw1 b--lm-orange segoe-bold tracked"
+      "secondary_button" ->
+        case background do
+          "dark" -> "f5 link dib ph3 pv2 br-pill lm-bg-dark-blue lm-white-hover b--lm-white-hover lm-orange pointer ba bw1 b--lm-orange segoe-bold tracked"
+          _ -> "f5 link dib ph3 pv2 br-pill lm-bg-white lm-white-hover lm-bg-dark-blue-hover lm-dark-blue pointer ba bw1 b--lm-dark-blue segoe-bold tracked"
+        end
```

Comment:
It may be good at some point to take out the css that is common to all buttons instead of repeating it.

Eg:

```elixir
"f5 link dib ..." <> case component do
  "primary_button" -> "fw6 ..."
  "secondary_button" ->
```

I think this part could be tidied up with more `get_class` functions and more pattern matching instead of using `case`